sdk/resourcehealth/Azure.ResourceManager.ResourceHealth/tests/autorest.tests.md

```diff
@@ -0,0 +1,13 @@
+# Generated code configuration
```

Comment:
I think we should remove this file.

Reply:
Also please confirm the sample project could build if we decide to introduce samples

Reply:
move to samples folder